sdcm/microbenchmarking.py

```diff
@@ -118,28 +131,36 @@ def get_diffs():
 
             last_val = get_metrica_val(list_of_results_from_db[last_idx])
             last_commit = get_commit_id(list_of_results_from_db[last_idx])
+            last_commit_date = get_commit_date(list_of_results_from_db[last_idx])
 
             best_result = get_best_result_for_metrica()
             best_result_val = get_metrica_val(best_result)
             best_result_commit = get_commit_id(best_result)
+            best_commit_date = get_commit_date(best_result)
 
             diff_last, diff_best = get_diffs()
 
             stats = {
                 "Current": cur_val,
-                "Last (commit id)": (last_val, last_commit),
-                "Best (commit id)": (best_result_val, best_result_commit),
+                "Last, commit, date": (last_val, last_commit, last_commit_date),
```

Comment:
Is this PR up to date?
I saw in the last email these were fixed to be: Last (commit,date)

Reply:
yes, this is the latest. I moved the prsentation of  this stat column to template. Template define, how display name of column